packages/openauth/src/authorizer.ts

```diff
@@ -561,6 +561,64 @@ export function authorizer<
     },
   )
 
+  app.post(
+    "/invalidate",
+    cors({
+      origin: "*",
+      allowHeaders: ["*"],
+      allowMethods: ["POST"],
+      credentials: false,
+    }),
+    async (c) => {
+      const form = await c.req.formData()
+
+      const refreshToken = form.get("refresh_token")
```

Comment:
According to the OAuth2 revokation spec, this should simply be 'token', and we can either determine the type from the token contents, or using an optional token_type_hint in the request https://datatracker.ietf.org/doc/html/rfc7009#section-2.1